src/porepy/numerics/interface_laws/cell_dof_face_dof_map.py

```diff
@@ -10,7 +10,7 @@
 import scipy.sparse as sps
 
 
-class EmptyDiscretization(object):
+class Cell_dof_face_dof_map(object):
```

Comment:
Also, the convention for class name would be CellDofFaceDofMap.  PEP0008.

The name is horrible, but very instructive. Also, it will likely not be much used.

Reply:
ok